django/db/models/base.py

```diff
@@ -1310,16 +1310,23 @@ def _check_default_pk(cls):
                 cls._meta.pk.remote_field.parent_link
             ) and
             not settings.is_overridden('DEFAULT_AUTO_FIELD') and
-            not cls._meta.app_config._is_default_auto_field_overridden
+            (
+                not cls._meta.app_config or
+                not cls._meta.app_config._is_default_auto_field_overridden
+            )
```

Comment:
I think we should ignore models with an invalid app label. I don't see much value in raising a warning for them.
```suggestion
            cls._meta.app_config and
            not cls._meta.app_config._is_default_auto_field_overridden
```